src/utils/barcode.ts

```diff
@@ -0,0 +1,60 @@
+import * as bwipjs from "bwip-js";
+import { sequenceS } from "fp-ts/lib/Apply";
+import { tryCatch2v } from "fp-ts/lib/Either";
+import { fromEither, taskEither, taskify } from "fp-ts/lib/TaskEither";
+
+export interface IBarcodeOutput {
+  png: string;
+  pngMimeType: string;
+  svg: string;
+  svgMimeType: string;
+}
+
+interface IBwipOptions {
+  bcid: string;
+  text: string;
+}
+
+const toBase64Svg = (options: IBwipOptions) =>
+  tryCatch2v(
+    () => {
+      const drawsvg = require("bwip-js/examples/drawing-svg");
```

Comment:
can we move this require call outside this method at top level ?

Reply:
It seems that It could not find a declaration file for this module, using `import * as drawsvg from "bwip-js/examples/drawing-svg";`

Reply:
it's ok to just move this outside the method at top level